libs/partners/upstage/langchain_upstage/layout_analysis.py

```diff
@@ -204,3 +204,45 @@ def lazy_load(self) -> Iterator[Document]:
                 exclude=self.exclude,
             )
             yield from parser.lazy_parse(blob)
+
+    def merge_and_split(
```

Comment:
Would suggest to stick to the public API of Base Loader. This looks a bit superfluous  -- i.e., can use a stand alone function or else accept a splitting or merging logic as part of the initializer to the logic.